build/config/compiler/BUILD.gn

```diff
@@ -228,9 +228,27 @@ config("runtime_default") {
   }
 }
 
+# To use different sanitizer options, use `gn args .` in the out folder and
+# use settings like:
+#
+#   is_clang=true
+#   is_debug=true
+#   optimize_for_size=false
+#   default_configs_sanitize = [ "//build/config/compiler:sanitize_address" ]
+#
 config("sanitize_default") {
```

Comment:
Can you do

```
declare_args() {
  # Enable AddressSanitizer.
  is_asan = false

  # Enable memory sanitizer.
  is_msan = false
}

config("sanitize_default") {
  configs = []

  if (is_asan) {
    configs = [":sanitize_address"]
  }

  if (is_msan) {
    configs = [":sanitizer_memory"]
  }
  }
}
```

Manually setting these lists on the command line should be an option of last resort for options we don't support or haven't thought of.

Reply:
I will do this in a separate PR - according to Boris what I was attempting to fix is not in the right place and we have a correct fix somewhere else. I will create a 'add sanitizer' PR that only does that. Thank you for the suggestion.